### PR TITLE
Update the ReadMe

### PR DESCRIPTION
**Pull request by asdonald**
# Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Checklist:

- [ ] All new and existing tests passed.
- [ ] The code follows the code style and conventions of this project.
- [ ] The change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

# Description of the change

...

# Motivation for the change

...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,141 +66,25 @@ at https://www.azureiotsuite.com.
...
-### Basic Deployment
-A lower-cost showcase solution best for discovering and learning about what a Remote Monitoring preconfigured solution has to offer.  *See [Basic vs. Standard Deployments](#basic-vs-standard-deployments) for more details.*
-```
-pcs --runtime java
-```
-
-### Local Deployment
-To set up a Remote Monitoring Solution locally see
-[Local Development](https://github.com/Azure/azure-iot-pcs-remote-monitoring-dotnet/wiki/Developer-Reference-Guide#local-development)
+You can follow along with the [Getting Started](https://docs.microsoft.com/azure/iot-suite/iot-suite-remote-monitoring-deploy) documentation to learn how to deploy using the web interface. You can also deploy using the [command line](https://docs.microsoft.com/azure/iot-suite/iot-suite-remote-monitoring-deploy-cli).
```
Comment (jillcary) [inline README.md]: formatting nit:
It might be good to break out the two options with bullet points instead of the paragraph format. Typically when I'm in a repo for the first time I either:
1) want to read a lot and read everything, or...
2) get me running as fast as possible 

Perhaps the format could be along the lines of:

## Deploy a solution
There are two ways to deploy a solution:
1) Deploy using the web interface using the instructions [here](https://docs.microsoft.com/azure/iot-suite/iot-suite-remote-monitoring-deploy).
2) Deploy using the [command line](https://docs.microsoft.com/azure/iot-suite/iot-suite-remote-monitoring-deploy-cli).

File: README.md [hunk 2]
```diff
@@ -66,141 +66,25 @@ at https://www.azureiotsuite.com.
...
-| 1 | [Azure IoT Hub][iot-hub] | S1 – Basic tier | Device management and communication |
-| 1 | [Azure Cosmos DB][cosmos-db] | Standard | Storing configuration data, and device telemetry like rules, alarms, and messages |
-| 5 | [Azure Storage Accounts][storage-account] | Standard GRS | 4 for VM storage, and 1 for the Stream Analytics checkpoints |
-| 1 | [Web Application][web-application] | | Hosting front-end web application |
-
-> Pricing information for these services can be found
-[here](https://azure.microsoft.com/pricing). Usage amounts and billing details
-for your subscription can be found in the
-[Azure Portal](https://portal.azure.com/).
+## All done? Connect a Physical device!
```
Comment (jillcary) [inline README.md]: Does "All done?" still apply with the addition of the simulated device instructions?

File: README.md [hunk 3]
```diff
@@ -66,141 +66,25 @@ at https://www.azureiotsuite.com.
...
-Microservices compartmentalize the code and provide well defined interfaces
-making the solution easier to understand and less monolithic. It also further
-expands options for partners that want to extend our current preconfigured
-solutions to build finished solutions that can be monetized.
-
-**Learn more about Docker Containers**
-* [Install Docker](https://docs.docker.com/engine/installation/)
-* [Common Docker Commands for Remote Monitoring](https://github.com/Azure/azure-iot-pcs-remote-monitoring-dotnet/wiki/Developer-Reference-Guide#common-docker-commands)
-* [Docker Getting Started](https://docs.docker.com/get-started/)
+[Learn more](https://docs.microsoft.com/azure/iot-suite/iot-suite-remote-monitoring-sample-walkthrough) about the remote monitoring architecture, including the use of microservices and docker containers.
```
Comment (jillcary) [inline README.md]: do we need to capitalize "Remote Monitoring?" or "Docker"

Comment (asdonald) [reply README.md]: Fixed!

File: README.md (hunk 2 above)
Comment (asdonald) [reply README.md]: Just removed all done.

File: README.md (hunk 1 above)
Comment (asdonald) [reply README.md]: Yes that looks much cleaner to me too

**Comment by dluc** [comment]
Seems we're dropping a lot of useful content. Are we moving that somewhere else?

File: README.md [hunk 4]
```diff
@@ -66,141 +66,27 @@ at https://www.azureiotsuite.com.
 Getting Started
 ===============
 
-## Prerequisites
-* [Azure Subscription](https://azure.microsoft.com/free/) (also see [permissions guidelines](https://docs.microsoft.com/azure/iot-suite/iot-suite-permissions))
```
Comment (dluc) [inline README.md]: this content seems useful, any particular reason for removing it?

File: README.md [hunk 5]
```diff
@@ -66,141 +66,27 @@ at https://www.azureiotsuite.com.
...
-
-## Setup
-1. Open a terminal window or command shell
-1. Check out the repository and submodules
-    ```
-    git clone --recursive https://github.com/Azure/azure-iot-pcs-remote-monitoring-java
-    ```
-    If you cloned the repo without the `--recursive` flag and it's your first time updating the submodules, you can use
-    `git submodule update --init --recursive`. More information on submodules [here](https://git-scm.com/book/en/v2/Git-Tools-    Submodules).
```
Comment (dluc) [inline README.md]: Users reaching the solution in GitHub typically want to work directly with the code, so this seems useful information, no?

File: README.md [hunk 6]
```diff
@@ -66,141 +66,27 @@ at https://www.azureiotsuite.com.
...
-
-### Basic Deployment
-A lower-cost showcase solution best for discovering and learning about what a Remote Monitoring preconfigured solution has to offer.  *See [Basic vs. Standard Deployments](#basic-vs-standard-deployments) for more details.*
-```
-pcs --runtime java
-```
-
-### Local Deployment
-To set up a Remote Monitoring Solution locally see
-[Local Development](https://github.com/Azure/azure-iot-pcs-remote-monitoring-dotnet/wiki/Developer-Reference-Guide#local-development)
```
Comment (dluc) [inline README.md]: same consideration. I think we should keep the GitHub READMEs oriented to developers, providing all the steps to work directly witht he code

File: README.md [hunk 7]
```diff
@@ -66,141 +66,27 @@ at https://www.azureiotsuite.com.
...
+## Deploy a solution
+There are two ways to deploy a solution:
+1) Deploy using the web interface using the instructions [here](https://docs.microsoft.com/azure/iot-suite/iot-suite-remote-monitoring-deploy).
+2) Deploy using the [command line](https://docs.microsoft.com/azure/iot-suite/iot-suite-remote-monitoring-deploy-cli).
 
 Common Scenarios
 ================
+## Create more simulated devices
+Once you have a solution up and running, you can [create more simluated devices](https://docs.microsoft.com/azure/iot-suite/iot-suite-remote-monitoring-test) to populate telemetry. You can then stop the default simulated devices by calling the simulation endpoint with the instructions [here](https://github.com/Azure/device-simulation-dotnet/wiki/%5BAPI-Specifications%5D-Simulations#stop-simulation).
```
Comment (dluc) [inline README.md]: typo: "simluated"

Comment (dluc) [reply README.md]: I would reword "populate telemetry"

File: README.md [hunk 8]
```diff
@@ -66,131 +66,27 @@ at https://www.azureiotsuite.com.
 Getting Started
```
Comment (jillcary) [inline README.md]: We should remove all mentions of "Preview" in the README as well.
